app/main.ipynb

```diff
@@ -25,18 +25,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import sys\n",
+    "from pickletools import float8\n",
```

Comment:
The import of `float8` from `pickletools` is unnecessary for the tasks described and should be removed to clean up the code.